### PR TITLE
Add ContainersReady status for Pods

### PR DESCRIPTION
**Pull request by lucperkins**
This PR addresses issue #9099. This PR is currently a WIP while I wait on information regarding appropriate usage.

### CODE REVIEW

File: content/en/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 1]
```diff
@@ -32,24 +32,14 @@ have a given `phase` value.
...
-
-* Failed: All Containers in the Pod have terminated, and at least one Container
-  has terminated in failure. That is, the Container either exited with non-zero
-  status or was terminated by the system.
-
-* Unknown: For some reason the state of the Pod could not be obtained, typically
-  due to an error in communicating with the host of the Pod.
+Value | Description
+:-----|:-----------
+`ContainersReady` | All Containers in the Pod are currently ready
```
Comment (tengqm) [inline content/en/docs/concepts/workloads/pods/pod-lifecycle.md]: No. This is not a Phase, it is a Condition.

File: content/en/docs/tutorials/stateful-application/cassandra.md [hunk 2]
```diff
@@ -95,65 +102,80 @@ The StatefulSet manifest, included below, creates a Cassandra ring that consists
...
+    ```
 
 {{< code file="cassandra/cassandra-statefulset.yaml" >}}
 
 ## Validating The Cassandra StatefulSet
 
 1. Get the Cassandra StatefulSet:
 
-       kubectl get statefulset cassandra
+    ```bash
```
Comment (tengqm) [inline content/en/docs/tutorials/stateful-application/cassandra.md]: This and the following indentations are breaking the numbered list. Please see preview result.

File: content/en/docs/concepts/workloads/pods/pod-lifecycle.md (hunk 1 above)
Comment (lucperkins) [reply content/en/docs/concepts/workloads/pods/pod-lifecycle.md]: Yep, you're right. I've fixed this and expanded the section on Pod conditions.

File: content/en/docs/tutorials/stateful-application/cassandra.md (hunk 2 above)
Comment (lucperkins) [reply content/en/docs/tutorials/stateful-application/cassandra.md]: The changes to this doc were included by accident. I've removed these changes from this PR.

**Comment by zacharysarah** [comment]
@lucperkins Thanks for taking a crack at this. Ultimately I'd like to see a more thorough pass for style guide consistency (for example, de-bullet and reorganize the _Pod conditions_ and _Example states_ sections), but this PR represents an iterative first step with formatting updates. 👍 

/lgtm
/approve

**Pull request merged**